pages/collaboration/team.mdx

```diff
@@ -0,0 +1,35 @@
+# Team Collaboration
+
+After having signed up to ModelZ, you’ve automatically received a personal account that you can use to run and deploy personal models. To deploy collaborate with others, however, you can create or join additional teams.
+
+## Create a Team
+
+You can create a team, and fill in your team member's e-mail address and role. 
+- If the email of team member is already signed up in ModelZ, the member will receive a team invitation by e-mail. After accepting the invitation, he can see the team in his team list.
+- If email is not signed up in ModelZ, the member will receive a sign up email with invitation.
+![create_team](./images/create_team.jpg)
+
+## Invite Team member
+
+To invite a new team member, first switch your user setting to team.
+![invite_team_member](./images/invite_team_member.jpg)
+
+
+## Team ID and APIKey
+
+Each team has its own team id and apikey, which a team member can obtain in the following way, using SDK or CLI for deployment control and endpoint access.
+![team_id_apikey](./images/team_id_apikey.jpg)
+
+
+## Role
+Teams have two different role with different levels of access privileges:
+- Owner
+- Member
+
+The user that creates a team is automatically set as the Owner for that team. The owner can assign any other roles within the team, as well as pending other members of the workspace.
```

Comment:
```suggestion
The user that creates the team is automatically set as the `Owner` of the team. The owner can assign any other roles within the team, as well as pending other members of the workspace.
```

What does "pending" here mean?

Reply:
Pending means the invited user not accept the invitation or the owner doesn't want the member to work on the team's resources anymore.

Reply:
But "pending" is not a verb, nor a -ing here.